packages/design-system/src/utils/event-bus.ts

```diff
@@ -33,6 +71,15 @@ export function createEventBus(): EventBus {
 		return () => off(eventName, fn);
```

Comment:
not sure why we return this function. it doesn't seem to be used. this is also not how any other EventEmitter implementation works (that I know off).

Reply:
That's a quite common pattern for providing a way to dispose a resource. But since we are not using it anywhere I think we can remove it